scenarios/19c_The_Oracle.cfg

```diff
@@ -72,17 +72,54 @@
         side=2
         # Spearman = fighter, Bowman = archer, Peasant = fighter, Swordsman = fighter, Mage = mixed fighter, City Militia = fighter
         recruit=Spearman,Bowman,Peasant,Swordsman,Mage,City Militia
+        extra_recruit={ON_DIFFICULTY4 (White Mage) (Javelineer) (Sergeant) (Giant Rat)}
         {INCOME4 6 5 4 3}
         {GOLD4 140 130 120 110}
         [modifications]
             {TRAIT_LOYAL OVERLAY=""}
             {TRAIT_RESILIENT}
             {TRAIT_FEARLESS}
+            [object]
+                id=M_R_healing_skill
+                duration=scenario
+                silent=yes
+                name= _ "Healing skill"
+                image=items/ring-red.png
+                description= _ "This unit has learned an odd style of healing skill: it can cure poison and heal a few HP of wounded friendly units."
```

Comment:
```suggestion
                description= _ "This unit has learned an odd style of healing: it can cure poison and heal a few hitpoints of wounded friendly units."
```